package.json

```diff
@@ -69,7 +70,8 @@
     "sinon-sandbox": "^1.0.2"
   },
   "peerDependencies": {
-    "react": ">=0.14"
+    "react": ">=0.14",
+    "prop-types": ">=15.5.6"
```

Comment:
`prop-types` should be a normal dependency, not a peer dependency, since it holds no state and doesn't need to be a singleton.